support/packages/github-to-jira-synchronization-tool/README.md

```diff
@@ -0,0 +1,46 @@
+# Github to Jira synchronization tool
+
+This project aim to be a replacement for Exalate, to update jira issues based on the state of the github issues. It is a node app that leverages github webhooks to do so.
+
+# Install the tool
+
+More information in how to configure this repos, deploy and use this tool can be found in [this confluence page](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/1954447876/Github+to+Jira+synchronization+tool)
+
+# How it works
+
+This tool leverage Github webhook, specificially issues and issue_comments. Each event is handled by a different JS file, all the event handlers can be found [here](https://github.com/liferay/liferay-frontend-projects/tree/98aef031f203f1718fdc5bdd05449b7410b304e2/support/packages/github-to-jira-synchronization-tool/src/event-handlers).
+
+It is implemented as a node http server, that exposes the endpoint `/api/github/webhooks` where it processes each webhook event.
+
+## Retries
+
+Since Jira may be down, but Github not. We need to make sure that those events that reach the server while Jira is down are retried later. Each time that a webhook event handler fails by a server error, the webhook is stored in a file. Every hour the file is checked and try to proccess the event again.
```

Comment:
Github => GitHub

that those events that reach => remove `those`

by a server error => because of a server error

and try to process => and it tries to process